### PR TITLE
PA-469 add api swagger content 

### PR DESCRIPTION
**Pull request by Pauljgraham**

### CODE REVIEW

File: src/TrackProgress/SFA.DAS.TrackProgress.Api/Controllers/TrackProgressController.cs [hunk 1]
```diff
@@ -36,6 +36,24 @@ public string Ukprn
...
+    ///
+    /// The overall progress of this apprenticeship will be constructed from these submissions. 
+    /// </remarks>
+    /// <param name="uln">The apprentice's Unique Learner Number.</param>
+    /// <param name="plannedStartDate">The planned start date for this apprenticeship.</param>
+    /// <param name="progress">The taxonomy content.Accepts an array of KSB progress percentiles.
+    /// 
+    /// The "id" field must be the GUID identifier of the KSB for the apprenticeship's course.
+    /// 
+    /// The "value" field must be the percentile range 1-100, denoting what percentage of the KSB has been completed.
```
Comment (benagain) [inline src/TrackProgress/SFA.DAS.TrackProgress.Api/Controllers/TrackProgressController.cs:54]: In the last few hours the range has been changed to 0-100 (not 1-100).

However, there will be other places that need to be updated, so let's do them all together in a different PR.

**Review by benagain — DISMISSED** [review]

File: src/TrackProgress/SFA.DAS.TrackProgress.Api/Controllers/TrackProgressController.cs (hunk 1 above)
Comment (Pauljgraham) [reply src/TrackProgress/SFA.DAS.TrackProgress.Api/Controllers/TrackProgressController.cs:54]: OK, I'll create one if you like

**Review by Chazzysb — APPROVED** [review]
Approved

**Review by Chazzysb — APPROVED** [review]
Approved
